common/views/components/ButtonOutlinedLink/ButtonOutlinedLink.tsx

```diff
@@ -50,7 +50,7 @@ const ButtonOutlinedLink = ({
         aria-expanded={ariaExpanded}
         onClick={handleClick}
         isOnDark={isOnDark}
-        href={isNextLink ? undefined : link}
+        href={isNextLink ? undefined : link.toString()}
```

Comment:
This felt pretty weird to me – I'm fairly confident there's a better way to do this.

Essentially at this point, we know that `link` is a string already (because it will be if it's not a `NextLink`), but TS still didn't like it unless I `.toString()`-ed it